### ISSUE TITLE
Possible to add these directly to muOS?

### ISSUE THREAD

**Issue by pete-woods** [mugwomp93/muOS_Customization#2]
These overlays are really nice - is it possible to add them to muOS directly so folks don't have to install them?

I don't mind raising pull requests there, but I didn't want to do it without getting your permission first.

**Comment by mugwomp93** [comment]
Thanks! Sure, I don't mind if you want to make a pull request. I'm not sure if or how they would handle the settings and custom palettes, but I don't have a problem with it.

**Comment by mugwomp93** [comment]
I'm also happy to chat with the muOS folks if they want to discuss. I'm subscribed to the muOS Discord; I just haven't been on it much recently.

**Comment by pete-woods** [comment]
Amazing, thank you for doing this. I'm sorry I got distracted and didn't reply sooner.

**Comment by mugwomp93** [comment]
Yeah, no worries! I'd hope this wasn't a life priority. Just let me know if/when you need anything from me.

**Comment by pete-woods** [comment]
Can I ask you to link the PR/commit, so I can understand better how to do this kinda thing myself in future?

**Comment by mugwomp93** [comment]
Unfortunately, I have no idea how to go about doing things properly in Github. I only post here because I saw it was where others had uploaded their files and Reddit doesn't block the links. I would guess that you could probably just ask the question like you did here, but I really don't know.

**Comment by pete-woods** [comment]
FYI, I got a PR merged here: https://github.com/MustardOS/internal/pull/424 to add your overlays to muOS directly.

You can see them here: https://github.com/MustardOS/internal/tree/main/init/MUOS/retroarch/overlays/mugwomp93

**Comment by mugwomp93** [comment]
Huh, interesting - thanks! Did you need to upload the directory,  or just make a request?

**Comment by pete-woods** [comment]
I used a Git client (in my case Fork.app) to clone the Git repo and commit the contents of your zip files into the overlay directory.

Then forked the repo on GitHub and raised a PR. Very standard kinda GitHub workflow.

If you ever want help updating your overlays in muOS don't hesitate to reach out to me at email@pete-woods.com or `pete-woods` on the muOS discord.

**Comment by mugwomp93** [comment]
Thanks, I do appreciate it, and I may take you up on that offer. The workflow you describe almost sounds like it's not gibberish to someone without a coding background who just likes to mess around with image editing.

Hopefully there won't be any updates to these unless it's something new (I see you got the GBP overlays I just fixed, which is convenient).

I have some more GB integer-scale overlays I'm just about done and I have a CubeXX that will hopefully be showing up soon - I imagine I'll probably start making overlays for that, too. So I could see wanting to add more than just the Perfect set in the future.

Incidentally, if you're not aware of them, you may want to check out 1playerinsertcoin's other Perfect Overlays (CRT, GG, SNES-specific CRT, and they're working on an NGPC overlay). They're all linked off their Reddit profile, though some have been added to the Onion git.

**Comment by pete-woods** [comment]
Good to know, thanks!
